src/Sentry.Profiling/Sentry.Profiling.csproj

```diff
@@ -5,14 +5,12 @@
     <TargetFrameworks>net6.0</TargetFrameworks>
     <PackageTags>$(PackageTags);Profiling;Diagnostic</PackageTags>
     <Description>Performance profiling support for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
-    <IsPackable>false</IsPackable>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Sentry\Sentry.csproj" />
-
     <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="0.2.421201" />
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.2" />
+    <ProjectReference Include="../../modules/perfview/src/TraceEvent/TraceEvent.csproj" />
```

Comment:
I imagine we'll need to figure out distributing this, also OK on a follow up PR since profilign isn't packaged yet

Reply:
yes, that was my thinking too